README.md

```diff
@@ -59,7 +59,13 @@ want to start reading from.
 
 ```javascript
 state.length //=> 0;
-state.get(0) //=> null;
+let record = state.get(0);
```

Comment:
I am proposing `state.get(0)` which is unrequested returns a Record Object, instead of `null`